### DISCUSSION TITLE
airgap ISO soup also updates the rules ?

### DISCUSSION THREAD

**Discussion by sinnetll19** [Security-Onion-Solutions/securityonion#3087] [Unsupported Versions] [answered]
I would like to know if when you mount an ISO and run soup to update your security onion grid in an airgap environment, does that update the rule-sets for the different tools as well? ex, does the ISO contain the latest suricata, sigma, yara rules etc.

**Comment by TOoSmOotH** [comment, marked as answer]
Yes it will update all the rules to the latest when you go to 2.3.30.

**Reply by sinnetll19** [reply]
thank you!
